src/topology/sheaves/presheaf_of_functions.lean

```diff
@@ -67,7 +67,8 @@ def CommRing_yoneda : TopCommRing.{u} ⥤ (Top.{u}ᵒᵖ ⥤ CommRing.{u}) :=
   { obj := λ X, continuous_functions X R,
     map := λ X Y f, continuous_functions.pullback f R },
   map := λ R S φ,
-  { app := λ X, continuous_functions.map X φ } }
+  { app := λ X, continuous_functions.map X φ, },
+  map_comp' := by { intros X Y Z f g, ext V h, dsimp, refl, }, }
```

Comment:
This proof still works `by tidy` and could be omitted, but the whole declaration is up against the `-T100000` limit, and now apparently slightly over, so I've put the `map_comp` proof in by hand.

It would be better to improve the `simp` lemmas so this `refl` isn't even needed, but not today.